EventsExpress/ClientApp/src/components/helpers/action-helpers.js

```diff
@@ -16,14 +16,14 @@ export const buildValidationState = async (responseData) => {
     return result;
 }
 
-export const getErrorMessege = async (responseData) => {
+export const getErrorMessage = async (responseData) => {
```

Comment:
This function is exported. Are you sure there are no other references to it after rename?